packages/api/test/e2e/promise-queries.spec.ts

```diff
@@ -35,7 +35,7 @@ describe.skip('e2e queries', () => {
   });
 
   it('subscribes to queries', (done) => {
-    api.rpc.chain.subscribeNewHead((error, header) => {
+    api.rpc.chain.subscribeNewHead((error: any, header: any) => {
```

Comment:
This should be adapted, interface has changed. (Not part of this PR, nor changed here, but as it stands the e2e test is broken)